sig/http/http_requester.rbs

```diff
@@ -0,0 +1,17 @@
+module Algolia
+  module Http
+    class HttpRequester
+      attr_accessor adapter: untyped
+
+      attr_accessor logger: untyped
+
+      def initialize: (untyped adapter, untyped logger) -> untyped
```

Comment:
`initialize` seems to return a `bool`

Reply:
While that's true, we should use `void` with the initialize method; this indicates that the return value should not be used (which is the case with every `initialize` method, since we never call it directly. Updating it in all files 👍

Reply:
oooh interesting! thanks!